### DISCUSSION TITLE
Local documentation

### DISCUSSION THREAD

**Discussion by katzuv** [pester/Pester#2373] [Q&A] [answered]
Hey,
I'd like to build a local copy of the documentation, where can I find the "source" files?

Thanks in advance!

**Comment by fflaten** [comment, marked as answer]
Hi. All commands in the module have built-in help. The website source is here https://github.com/pester/docs, including a devcontainer to help you with all prereqs to build it.

**Reply by nkrishnakishor** [reply]
Thanks for the reply, I checked the doc link and it has information on parameters of a function but cannot find for the PowerShell script itself.

**Reply by fflaten** [reply]
Which script?
